docs/source/guides/understanding_types_and_tags.ipynb

```diff
@@ -383,13 +375,13 @@
    "source": [
     "### Index and Time Index Tags\n",
     "\n",
-    "When creating a DataTable, you have the option to specify which column represents the index and which column represents the time index. If these columns are specified, semantic tags of `index` and `time_index` are applied to the specified columns. Behind the scenes, Woodwork is performing additional validation checks on the columns to make sure they are appropriate. For example, index columns must be unique, and time index columns must contain datetime values or values that can be converted to datetimes.\n",
+    "When initializing Woodwork, you have the option to specify which column represents the index and which column represents the time index. If these columns are specified, semantic tags of `index` and `time_index` are applied to the specified columns. Behind the scenes, Woodwork is performing additional validation checks on the columns to make sure they are appropriate. For example, index columns must be unique, and time index columns must contain datetime values or numeric values.\n",
     "\n",
-    "Because of the need for these validation checks, you can't set the `index` or `time_index` tags directly on a column. In order to designate a column as the index, the `set_index` method should be used. Similarly, in order to set the time index column, the `set_time_index` method should be used. Optionally, these can be specified when initially creating the DataTable.\n",
+    "Because of the need for these validation checks, you can't set the `index` or `time_index` tags directly on a column. In order to designate a column as the index, the `set_index` method should be used. Similarly, in order to set the time index column, the `set_time_index` method should be used. Optionally, these can be specified when initializing Woodwork.\n",
     "\n",
     "#### Setting or changing the index\n",
     "\n",
-    "Create a new sample data set that contains columns that can be used as index and time index columns by using the `index` or `time_index` parameters."
+    "Create a new sample DataFrame that contains columns that can be used as index and time index columns by using the `index` or `time_index` parameters."
```

Comment:
It might be good to showcase an example of using the `index` and `time_index` parameters.

Reply:
Hmmm...the purpose of this guide is really to explain what logical types and semantic tags are and how they work, not necessarily to show the various ways to set the index. Having said that, I can see how this wording might be a little confusing since the parameters are mentioned and not used anywhere. I think there are a couple options:
- Update the wording to remove the reference to the parameters
- Leave the wording as is and update the example to use the parameters

Any preference on which path to take?

Reply:
Either path is fine but you might consider updating the header. I think "Setting or changing the index" kind of implies various ways of setting the index.